src/ui/components/Video/RecordedCursor.tsx

```diff
@@ -39,9 +39,18 @@ export function RecordedCursor() {
           element.style.top = `${mouseY}%`;
           element.style.transform = `scale(${cursorScale})`;
           element.style.setProperty("--click-display", shouldDrawClick ? "block" : "none");
+          // Testing support
+
+          element.dataset.cursordisplay = "true";
+          element.dataset.clickdisplay = shouldDrawClick ? "true" : "false";
+          element.dataset.clientx = `${mouseEvent.clientX}`;
+          element.dataset.clienty = `${mouseEvent.clientY}`;
         } else {
           element.style.display = "none";
           element.style.setProperty("--click-display", "none");
+
+          element.dataset.cursordisplay = "false";
+          element.dataset.clickdisplay = "false";
```

Comment:
Maybe it doesn't matter too much but should we reset the data client x and y values to 0 in this case?